components/esp_lvgl_port/include/esp_lvgl_port.h

```diff
@@ -59,13 +62,15 @@ typedef struct {
     } flags;
 } lvgl_port_display_cfg_t;
 
+#if __has_include ("esp_lcd_touch.h")
 /**
  * @brief Configuration touch structure
  */
 typedef struct {
     lv_disp_t *disp;    /*!< LVGL display handle (returned from lvgl_port_add_disp) */
     esp_lcd_touch_handle_t   handle;   /*!< LCD touch IO handle */
 } lvgl_port_touch_cfg_t;
+#endif
 
 /**
  * @brief Touch IO configuration structure
```

Comment:
Not related to your changes, but could you please update this comment?

Reply:
Thank you. Fixed.